relay-server/src/managed/managed.rs

```diff
@@ -135,21 +145,64 @@ impl<T: Counted> Managed<T> {
         self.meta.scoping
     }
 
-    pub fn split<F, I, S>(self, mut f: F) -> Split<I::IntoIter, I::Item>
+    /// Splits [`Self`] into two other [`Managed`] items.
+    ///
+    /// The two resulting managed instances are expected to have the same outcomes.
```

Comment:
Maybe I'm being pedantic but I feel like this can be misread as "The two resulting managed instances are expected to have the same outcomes _as each other_" or "The two resulting managed instances are _each_ expected to have the same outcomes _as the original instance_". 
```suggestion
    /// The two resulting managed instances together are expected to have the same outcomes as the original instance..
```